### ISSUE TITLE
test_fuzz: stray process left behind

### ISSUE THREAD

**Issue by jwilk** [jwilk/python-afl#4]
Sometimes `test_fuzz.py` leaves behind a stray process:

```console
$ nosetests tests/test_fuzz.py
....
----------------------------------------------------------------------
Ran 4 tests in 0.533s

OK

$ ps -u $(whoami) -o pid,state,cmd | grep ' [T] '
7607 T /usr/bin/python /home/jwilk/python-afl/tests/target_persistent.py
```

CC: @danstender

**Comment by danstender** [comment]
I would have guessed it's the tests. Thx for getting into it.
Dan

**Comment by jwilk** [comment]
Simpler reproducer:
```console
$ mkdir in out
$ echo moo > in/moo
$ timeout 1 py-afl-fuzz -i in -o out python tests/target_persistent.py
```

**Comment by jwilk** [comment]
It's a bug in AFL proper:
https://groups.google.com/d/topic/afl-users/E37s4YDti7o

I'll try to implement a work-around in the test suite.

**Comment by jwilk** [comment]
Work-around implemented in 6d6bac7bade633d9a898629e44e97c2709d5ba64.

I'll make a new release soon.

**Comment by jwilk** [comment]
I've just released python-afl 0.5.5, which includes the aforementioned work-around.
